README.md

```diff
@@ -71,6 +71,16 @@ choco install saml2aws
 saml2aws --version
 ```
 
+### Linux
+
+#### Void Linux
+
+If you are on Void Linux you can use xbps to install the saml2aws package!
```

Comment:
Can you make Void Linux a link to the project?

Reply:
Sure, it's done.